frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -151,118 +129,137 @@ class PopupCreateFile extends React.Component {
     })
   }
 
-  postFile = async (file) => {
-    const { state, props } = this
+  filesAlreadyInListFilter = (newFile, listToFilter) => listToFilter.some(stateFile => stateFile.name === newFile.name)
```

Comment:
this function returns a boolean, it's name should start with "is", "should", "has" or something that implies that it returns a boolean.
This function name seems to return a list